mocks/ileMock.js

```diff
@@ -0,0 +1 @@
+export default '';
```

Comment:
I believe this filename is supposed to be `fileMock.js` not `ileMock.js`

Reply:
> I believe this filename is supposed to be `fileMock.js` not `ileMock.js`

Thanks. i'll fix that.